### DISCUSSION TITLE
X values wrong ellipsize

### DISCUSSION THREAD

**Discussion by rezafaraji93** [patrykandpatrick/vico#392] [Questions] [answered]
I have x values of size 30, and I limited it to show only 3 values. now It has applied ellipsize to the first and last value in the chart which I do not want the view like this:

![Screenshot 2023-09-07 at 12 43 57 PM](https://github.com/patrykandpatrick/vico/assets/71515156/46cb71b9-160a-408c-acfa-f147f5690b24)

**Comment by rezafaraji93** [comment]
Any ideas?

**Reply by patrickmichalik** [reply]
Hello! It looks like there isn’t enough room for the two labels in question. Are you using `HorizontalLayout.Segmented` (the default) or `HorizontalLayout.FullWidth`?

**Reply by rezafaraji93** [reply]
I am using `HorizontalLayout.FullWidth`.

**Reply by patrickmichalik** [reply]
Got it. If so, you can add some horizontal padding to make room for the labels. We plan on adding a feature that will automatically add the required amount of padding.

**Reply by patrickmichalik** [reply, marked as answer]
[Vico 1.13.0 Alpha 9](https://github.com/patrykandpatrick/vico/releases/tag/v1.13.0-alpha.9) introduces the feature mentioned above.
